### PR TITLE
feat: permission prompt for new article

### PR DESCRIPTION
**Pull request by el**
## Changes

Screenshot 2022-12-09 at 17 44 12

### Describe what this PR does
- Short and concise, bullet points can help
- Screenshots if applicable can also help

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/notifications/EnableNotification.tsx [hunk 1]
```diff
@@ -78,12 +78,15 @@ function EnableNotification({
       </div>
     );
   }
+  const message =
+    source === NotificationPromptSource.NewSource
+      ? 'Would you like to get notified on the status of your article submissions in real time?'
+      : `Want to get notified when ${
+          parentCommentAuthorName ?? 'someone'
+        } responds so you can continue the conversation?`;
```
Comment (rebelchris) [inline packages/shared/src/components/notifications/EnableNotification.tsx:86]: Prefer this to be a objectMap function if we can.
```
const message: <Type, string> = { 
NewSource: ''
Comment: ''
```

**Review by rebelchris — CHANGES_REQUESTED** [review]
Happy with it, just one suggestion to map the text as it might grow and overall cleaner I think.
Let me know what you think 👍

File: packages/shared/src/components/notifications/EnableNotification.tsx (hunk 1 above)
Comment (el) [reply packages/shared/src/components/notifications/EnableNotification.tsx]: Done 👍

**Review by rebelchris — APPROVED** [review]
So fast! 🦜

**Pull request merged**